assets/js/components/higherorder/withdata.js

```diff
@@ -265,7 +265,7 @@ const withData = (
 
 			// If we have an error, display the DataErrorComponent.
 			if ( error ) {
-				return ( 'string' !== typeof error ) ? error : getDataErrorComponent( moduleName, error, layoutOptions.inGrid, layoutOptions.fullWidth, layoutOptions.createGrid );
+				return ( 'string' !== typeof error ) ? error : getDataErrorComponent( moduleName, error, layoutOptions.inGrid, layoutOptions.fullWidth, layoutOptions.createGrid, data );
```

Comment:
This function is also called in various Site Kit module components, where the error object (here `data`) also needs to be passed, otherwise the link to fix the issue won't appear.

Reply:
This one is already done by @felixarntz. Resolving this message.